go.mod

```diff
@@ -28,12 +28,12 @@ require (
 	k8s.io/utils v0.0.0-20190809000727-6c36bc71fc4a
 	sigs.k8s.io/cluster-api v0.2.0
 	sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm v0.1.0
-	sigs.k8s.io/controller-runtime v0.2.0
+	sigs.k8s.io/controller-runtime v0.2.2
 	sigs.k8s.io/yaml v1.1.0
 )
 
 replace (
 	k8s.io/api => k8s.io/api v0.0.0-20190704095032-f4ca3d3bdf1d
-	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190704094733-8f6ac2502e51
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20190918200256-06eb1244587a
```

Comment:
Why has this changed?

Reply:
I guess we don't need it anymore? I don't really understand go.mod if I'm honest

Reply:
We should definitely keep the apimachinery one, otherwise it might break downstream consumers. 

Do you want to update client-go though? Because that also might break 😬

Reply:
Downstream consumers don't look at `replace` directives, so I think we'll be okay.

I need client-go > 1.15 because that's where the event manager configuration was added.

Reply:
I see. Let's make sure this doesn't impact other things as well. The versions we're using here collide with the ones specified in controller-runtime and that has caused trouble in the past

Reply:
worked fine in dev :woman_shrugging: